Servers/UI/OJS.Servers.Ui.Infrastructure/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -45,23 +45,7 @@ public static class ServiceCollectionExtensions
             IConfiguration configuration)
         {
             services
-                .Configure<DistributorConfig>(configuration.GetSection(nameof(DistributorConfig)))
-                .ValidateLaunchSettings();
-
-            return services;
-        }
-
-        private static IServiceCollection ValidateLaunchSettings(this IServiceCollection services)
-        {
-            var requiredConfigValues = new[]
-            {
-                EnvironmentVariables.DistributorBaseUrlKey,
-                EnvironmentVariables.ApplicationUrl,
-                EnvironmentVariables.RedisConnectionString,
-                EnvironmentVariables.SharedAuthCookieDomain,
-            };
-
-            EnvironmentUtils.ValidateEnvironmentVariableExists(requiredConfigValues);
```

Comment:
Now we are not making any validations in UI?

Reply:
The validation for UI are the same as in the Administration. That's why they are at one place. The main difference is the Youtube environment which we already discussed.

Reply:
Where is the EnvironmentUtils.ValidateEnvironmentVariableExists() called in UI?

Reply:
I deleted them unconsciously... I brought them back in ServiceCollectionExtensions in the UI.